src/mlpack/tests/main_tests/krann_test.cpp

```diff
@@ -0,0 +1,335 @@
+#include <string>
```

Comment:
Could you add a header comment and the license to this file please?  You can use any other test file as an example. :)